pyproject.toml

```diff
@@ -32,8 +32,11 @@ actfw-core = "2.0.0"
 result = "^0.6.0"
 
 [tool.poetry.dev-dependencies]
-Sphinx = "^3.5.3"
-numpy = "^1.19"
+Sphinx = "^4.3.0"
+numpy = [
+    {version = "^1.19", python = "<3.7"},
+    {version = "^1.21.3", python = ">=3.7,<3.11"}
+]
```

Comment:
numpy 1.21.3 supports python 3.7-3.10, and is the minimum version supporting 3.10.